indico/modules/events/registration/fields/simple.py

```diff
@@ -330,3 +341,16 @@ def _to_machine_date(date):
 
 def _to_date(date):
     return datetime.strptime(date, '%Y-%m-%d').date()
+
+
+class MultiChoiceField(ChoiceBaseField):
+    name = 'multi_choice'
+    wtf_field_class = wtforms.SelectMultipleField
+
+    def get_friendly_data(self, registration_data):
+        reg_data = registration_data.data
+        return [self.form_item.data['captions'][x] for x in reg_data]
+
+    def calculate_price(self, registration_data):
+        reg_data = [x for x in self.form_item.versioned_data['choices'] if x['id'] in registration_data.data]
+        return sum([item['price'] for item in reg_data if item['is_billable']])
```

Comment:
no need for the `[]` - sum accepts a generator:

    return sum(item['price'] for item in reg_data if item['is_billable'])